### ISSUE TITLE
mark-callable-contracts rule is firing a false positive on constants

### ISSUE THREAD

**Issue by habdelra** [protofire/solhint#118]
The mark-callable-contracts rule is being triggered against emitting events, as events are camel-cased very similarly to contracts. We should update this rule so that it does not get triggered by emitting an event. Take for instance this example:
```
contract A {
  uint8 private constant TOKEN_DECIMALS = 15;

  function b() public view returns(uint8) {
    return TOKEN_DECIMALS;
  }

}
```
